lib/concat/concat.test.js

```diff
@@ -0,0 +1,24 @@
+var concat = require('.')
+
+const arr = [1, 2, 3, 4, 5];
+const str = 'hello';
+
+test("concats array with any number of arguments", () => {
+  expect(concat(arr, 'a')).toEqual([...arr, 'a'])
+  expect(concat(arr, 'a', 'b')).toEqual([...arr, 'a', 'b'])
+  expect(concat(arr, 'a', 'b', 'c')).toEqual([...arr, 'a', 'b', 'c'])
+  expect(concat(arr, 'a', 'b', 'c', 'd')).toEqual([...arr, 'a', 'b', 'c', 'd'])
```

Comment:
doesn't concat alway return a string?

Reply:
According to [ramda concat](http://ramdajs.com/docs/#concat) - not always

Reply:
If both arguments are strings, the result will be a string, right